stl/inc/__msvc_format_ucd_tables.hpp

```diff
@@ -364,6 +375,168 @@ inline constexpr _Unicode_property_data<_Extended_Pictographic_property_values,
+    0xaab2, 0xaab5, 0xaab7, 0xaab9, 0xaabe, 0xaac0, 0xaac1, 0xaac2, 0xaaec, 0xaaee, 0xaaf6, 0xaaf7, 0xabe5, 0xabe6,
+    0xabe8, 0xabe9, 0xabed, 0xabee, 0xfb1e, 0xfb1f, 0xfe00, 0xfe10, 0xfe20, 0xfe30, 0xff9e, 0xffa0, 0x101fd, 0x101fe,
+    0x102e0, 0x102e1, 0x10376, 0x1037b, 0x10a01, 0x10a04, 0x10a05, 0x10a07, 0x10a0c, 0x10a10, 0x10a38, 0x10a3b, 0x10a3f,
+    0x10a40, 0x10ae5, 0x10ae7, 0x10d24, 0x10d28, 0x10eab, 0x10ead, 0x10efd, 0x10f00, 0x10f46, 0x10f51, 0x10f82, 0x10f86,
+    0x11001, 0x11002, 0x11038, 0x11047, 0x11070, 0x11071, 0x11073, 0x11075, 0x1107f, 0x11082, 0x110b3, 0x110b7, 0x110b9,
+    0x110bb, 0x110c2, 0x110c3, 0x11100, 0x11103, 0x11127, 0x1112c, 0x1112d, 0x11135, 0x11173, 0x11174, 0x11180, 0x11182,
+    0x111b6, 0x111bf, 0x111c9, 0x111cd, 0x111cf, 0x111d0, 0x1122f, 0x11232, 0x11234, 0x11235, 0x11236, 0x11238, 0x1123e,
+    0x1123f, 0x11241, 0x11242, 0x112df, 0x112e0, 0x112e3, 0x112eb, 0x11300, 0x11302, 0x1133b, 0x1133d, 0x1133e, 0x1133f,
+    0x11340, 0x11341, 0x11357, 0x11358, 0x11366, 0x1136d, 0x11370, 0x11375, 0x11438, 0x11440, 0x11442, 0x11445, 0x11446,
+    0x11447, 0x1145e, 0x1145f, 0x114b0, 0x114b1, 0x114b3, 0x114b9, 0x114ba, 0x114bb, 0x114bd, 0x114be, 0x114bf, 0x114c1,
+    0x114c2, 0x114c4, 0x115af, 0x115b0, 0x115b2, 0x115b6, 0x115bc, 0x115be, 0x115bf, 0x115c1, 0x115dc, 0x115de, 0x11633,
+    0x1163b, 0x1163d, 0x1163e, 0x1163f, 0x11641, 0x116ab, 0x116ac, 0x116ad, 0x116ae, 0x116b0, 0x116b6, 0x116b7, 0x116b8,
+    0x1171d, 0x11720, 0x11722, 0x11726, 0x11727, 0x1172c, 0x1182f, 0x11838, 0x11839, 0x1183b, 0x11930, 0x11931, 0x1193b,
+    0x1193d, 0x1193e, 0x1193f, 0x11943, 0x11944, 0x119d4, 0x119d8, 0x119da, 0x119dc, 0x119e0, 0x119e1, 0x11a01, 0x11a0b,
+    0x11a33, 0x11a39, 0x11a3b, 0x11a3f, 0x11a47, 0x11a48, 0x11a51, 0x11a57, 0x11a59, 0x11a5c, 0x11a8a, 0x11a97, 0x11a98,
+    0x11a9a, 0x11c30, 0x11c37, 0x11c38, 0x11c3e, 0x11c3f, 0x11c40, 0x11c92, 0x11ca8, 0x11caa, 0x11cb1, 0x11cb2, 0x11cb4,
+    0x11cb5, 0x11cb7, 0x11d31, 0x11d37, 0x11d3a, 0x11d3b, 0x11d3c, 0x11d3e, 0x11d3f, 0x11d46, 0x11d47, 0x11d48, 0x11d90,
+    0x11d92, 0x11d95, 0x11d96, 0x11d97, 0x11d98, 0x11ef3, 0x11ef5, 0x11f00, 0x11f02, 0x11f36, 0x11f3b, 0x11f40, 0x11f41,
+    0x11f42, 0x11f43, 0x13440, 0x13441, 0x13447, 0x13456, 0x16af0, 0x16af5, 0x16b30, 0x16b37, 0x16f4f, 0x16f50, 0x16f8f,
+    0x16f93, 0x16fe4, 0x16fe5, 0x1bc9d, 0x1bc9f, 0x1cf00, 0x1cf2e, 0x1cf30, 0x1cf47, 0x1d165, 0x1d166, 0x1d167, 0x1d16a,
+    0x1d16e, 0x1d173, 0x1d17b, 0x1d183, 0x1d185, 0x1d18c, 0x1d1aa, 0x1d1ae, 0x1d242, 0x1d245, 0x1da00, 0x1da37, 0x1da3b,
+    0x1da6d, 0x1da75, 0x1da76, 0x1da84, 0x1da85, 0x1da9b, 0x1daa0, 0x1daa1, 0x1dab0, 0x1e000, 0x1e007, 0x1e008, 0x1e019,
+    0x1e01b, 0x1e022, 0x1e023, 0x1e025, 0x1e026, 0x1e02b, 0x1e08f, 0x1e090, 0x1e130, 0x1e137, 0x1e2ae, 0x1e2af, 0x1e2ec,
+    0x1e2f0, 0x1e4ec, 0x1e4f0, 0x1e8d0, 0x1e8d7, 0x1e944, 0x1e94b, 0xe0020, 0xe0080, 0xe0100, 0xe01f0};

```

Comment:
Given how simple these are to derive from the full Grapheme_cluster_break property this does seem like a waste of space. Since Grapheme_Extend is used to derive Grapheme_cluster_break then perhaps the solution to this is to use this table and derive the contents of the other. In any event I'm fine with that being left to a later PR.

Reply:
I filed #3696 for followup, thanks.